python/cugraph-service/client/cugraph_service_client/client.py

```diff
@@ -1507,9 +1514,26 @@ def excepthook(exc):
 
     @staticmethod
     def __get_vertex_edge_id_obj(id_or_ids):
-        # FIXME: do not assume all values are int32
+        # Force np.ndarray
+        if not isinstance(id_or_ids, (int, Sequence, np.ndarray)):
+            if not isinstance(cp, MissingModule) and isinstance(id_or_ids, cp.ndarray):
+                id_or_ids = id_or_ids.get()
+            elif not isinstance(cudf, MissingModule) and isinstance(
+                id_or_ids, cudf.Series
+            ):
+                id_or_ids = id_or_ids.values_host
+            elif not isinstance(pandas, MissingModule) and isinstance(
+                id_or_ids, pandas.Series
+            ):
+                id_or_ids = id_or_ids.to_numpy()
+
```

Comment:
This can be a `FIXME` to get CI unblocked ASAP, but should there be a closing `else:` block to raise an informative exception for the unlikely event that none of the missing modules are present?

Reply:
added one